src/extension.ts

```diff
@@ -4,4 +4,14 @@ import { sortPackageJson } from './sortPackageJson'
 
 export function activate(context: vscode.ExtensionContext) {
   context.subscriptions.push(vscode.commands.registerTextEditorCommand('editor.sortPackageJson', sortPackageJson))
+  context.subscriptions.push(vscode.languages.registerCodeActionsProvider({ language: 'json', pattern: '**/package.json' }, new CodeActionProvider()))
+}
+
+export class CodeActionProvider implements vscode.CodeActionProvider {
+  public provideCodeActions(): ProviderResult<vscode.Command[]> {
```

Comment:
You probably mean `vscode.ProviderResult` here.

You can run `npm run compile` or `npm test` to see the error.

Reply:
Oops, updated

Reply:
Should the tests / extension compilation be run as part of the GitHub Actions CI checks maybe?

Reply:
Yes. Typically that should be the case. It's been a while ago so I can't remember if it is because the vscode building doesn't work on the CI or I just haven't set it up.

I can take a look at it again after this PR is merged.